src/MicrosoftTeams.schema.json

```diff
@@ -5,22 +5,22 @@
     "properties": {
         "manifestVersion": {
             "type": "string",
-            "description": "The version of the manifest schema this manifest is using.",
+            "description": "The version of the schema this manifest is using.",
             "maxLength": 16
         },
         "version": {
             "type": "string",
-            "description": "The version of the manifest you are submitting. Changes to your manifest should cause a version change. This version string must follow the semver standard.",
+            "description": "The version of your app you are submitting. Changes to your manifest should cause a version change. This version string must follow the semver standard.",
             "maxLength": 256
         },
         "id": {
             "type": "string",
             "description": "A unique identifier for this set of tabs and bots. This id must be a guid.",
             "pattern": "^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$"
         },
-        "reverseDomainId": {
+        "packageName": {
```

Comment:
We need to add this into our telemetry pipeline. @ydogandjiev @richardtaylorrt  please correct me if i'm wrong.